src/product/product.controller.ts

```diff
@@ -0,0 +1,33 @@
+import { Body, Controller, Delete, Get, Param, Patch, Post } from '@nestjs/common';
+import { ProductService } from './product.service';
+import { CreateProductDto } from './dto/create-product.dto';
+import { Product } from './product.schema';
+import { UpdateProductDto } from './dto/update-product.dto';
+
+@Controller('product')
+export class ProductController {
+    constructor(private readonly productService:ProductService) {}
+
+    @Post('add')
+    async create(@Body() CreateDto: CreateProductDto){
+        return this.productService.create(CreateDto)
+    }
+
+    @Get('find')
+   async findAll(): Promise<Product[]> {
+        const products = this.productService.findAllProducts();
+        return products;
+    }
```

Comment:
this can directly return like the above function " return this.productService......"